app/api/auth.py

```diff
@@ -0,0 +1,182 @@
+from datetime import timedelta
+from app import db, jwt
+from flask import Blueprint, request, jsonify
+from flask_jwt_extended import create_access_token, jwt_required, get_jwt
+from app.models import Users, Account, TokenBlockList
+
+bp = Blueprint('auth', __name__)
+
+
+@bp.route('/signup', methods=['POST'])
+def signup():
+    try:
+        data = request.get_json()
+
+        attributes = data.get('data', {}).get('attributes', {})
+        username = attributes.get('username')
+        password = attributes.get('password')
+        account_name = attributes.get('account_name')
+
+        if not attributes or not username or not password or not account_name:
+            return jsonify({'errors': [{
+                'status': '422',
+                'detail': 'Some attributes are missing. '
+                'Username, password and account name are required'
+            }]}), 422
```

Comment:
If the user make some mistake, it is very hard to understand what exactly causes an error.